### DISCUSSION TITLE
better highlight current active endpoint

### DISCUSSION THREAD

**Discussion by flexdinesh** [Kong/insomnia#8116] [Feedback]
Could you please add an overlay on the background or do something to highlight the current active endpoint better? This has been my biggest pain point with insomnia for over a year now. I have a pretty big collection (close to 100 - not shown in the attached image) and I switch between the endpoints often. Everytime I need to figure which endpoint I'm on, it takes a while for my brain to spot it on the left bar because the current active endpoint highlight (think vertical bar on the left) is too subtle to grab attention. 

![CleanShot 2024-10-23 at 10 18 21@2x](https://github.com/user-attachments/assets/95a8238a-40f8-44e6-882a-68e6741dda63)

**Comment by natthia** [comment]
Hi, I believe tabs will be added in the next update. Maybe this is fixed by that?
In the meantime I think this can help you out with that: https://insomnia.rest/plugins/insomnia-plugin-request-navigator-forked
